src/Symfony/Component/HttpKernel/Tests/Fixtures/Controller/VariadicController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\Controller;
+
+class VariadicController
+{
+    public function action($foo, \stdClass ...$bar)
```

Comment:
Failure on HHVM: `Fatal error: Parameter $bar is variadic and has a type constraint (stdClass); variadic params with type constraints are not supported in /home/travis/build/symfony/symfony/src/Symfony/Component/HttpKernel/Tests/Fixtures/Controller/VariadicController.php on line 7`